src/components/mainMenu/mainMenu.css

```diff
@@ -0,0 +1,40 @@
+.swquiz-mainmenu {
+  color: rgba(0, 0, 0, 0.4);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background: rgba(255, 255, 255);
+  box-shadow: 0 4px 4px rgb(0, 0, 0, 0.25), 4px 4px 40px rgb(255, 0, 0, 0.9);
+  border-radius: 16px;
+  height: 11.25rem;
+}
+
+.swquiz-mainmenu-item {
+  font-size: 1.7rem;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+.swquiz-mainmenu-item.selected {
```

Comment:
Odpalilam Twoj kod w przegladarce i jak sie wybierze dany paragraf to sie ladnie podswietla, to co moglabym zasugerowac to zeby "people" bylo domyslnie zaznaczone moze, bo bez tego UI wydaje mi sie troche nieintuicyjne :)

Reply:
Done :)